### ISSUE TITLE
Add method progress

### ISSUE THREAD

**Issue by mpo-mjolner** [Open-Pectus/Open-Pectus#376]

**Comment by nneskildsf** [comment]
Good idea! Somehow show the progrss on the Unit Operaton card on the dashboard.

One way to do this is to show how much of method is completed in % or by text messages:
 * If there is more method to do: "Method in progress"
 * If method is exhausted: "Method completed"
